backend/src/router/init.go

```diff
@@ -296,6 +296,7 @@ func InitializeRouter() {
 	sessionsRouter.HandleFunc("", authMember(getSessions)).Methods("GET")
 	sessionsRouter.HandleFunc("/{id}", authMember(getSession)).Methods("GET")
 	sessionsRouter.HandleFunc("/{id}", authCoordinator(updateSession)).Methods("PUT")
+	sessionsRouter.HandleFun("/{id}", authAdmin(deleteSession)).Methods("DELETE")
```

Comment:
```suggestion
	sessionsRouter.HandleFunc("/{id}", authAdmin(deleteSession)).Methods("DELETE")
```
Probably a typo because the function `HandleFun` does not exist.